_posts/2016-05-13-initiative-circles.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+name: initiative-circles 
+title: Initiative Circles
+date: 2016-05-13 00:20:00 +00:00
+author: Mashooq Badar 
+image:
+   src: /assets/img/custom/blog/penguins.jpg
+tags:
+- codurance 
+- organisation
+---
+
+We have been trying to encourage decentralised decision making at Codurance. To that effect, everyone within the company understands that they have the power to make any decision they need to make so long as they seek advise from the people that may be impacted. In order to help pople make effective decisions, all company information is available to everyone in the company, including financials and salaries. This is all well and good, but in practice people do not simply go and start making informed strategic decisions just because they have the power to do so. For one, they are busy with their day jobs and simply do not have much time to look at the company holistically. Even if they have some wonderful ideas, they may not have the inclination or confidence to kick-off the initiative.
```

Comment:
Its up to you to give some background of where you got the idea or inspiration from ? Teal Companies ?

Would make an interesting read.